src/LfMerge.Core/MagicStrings.cs

```diff
@@ -94,6 +94,7 @@ static MagicStrings()
 
 		// Other magic strings that don't fall into any particular category
 		public const string UnknownString = "***";
+		public const string EnvVar_TrustToken = "LD_TRUST_TOKEN";
```

Comment:
Per Robin's comment above:
```suggestion
		public const string EnvVar_TrustToken = "LANGUAGE_DEPOT_TRUST_TOKEN";
```